### DISCUSSION TITLE
Adjust Dropdown Size

### DISCUSSION THREAD

**Discussion by DomenicoFormoso** [Megabit/Blazorise#5490] [General]
Is it possible to change the size of the dropdown component with Size="Size.ExtraSmall"?

**Comment by stsrki** [comment]
Yes, It is set on the DropdownToggle component.

**Comment by DomenicoFormoso** [comment]
I tried but it doesn't work with 1.52 with last version and fluent

**Reply by stsrki** [reply]
Ah, so the problem might be in the missing styles for the Fluent CSS. Can I ask you to open a new issue?

**Comment by DomenicoFormoso** [comment]
ok. tahnks
